bedrock/firefox/templates/firefox/accounts-2018.html

```diff
@@ -38,7 +38,8 @@ <h1 class="mzp-c-hero-title">{{ _('One log-in. Power and privacy everywhere.') }
       {% endif %}
       <div id="hero-cta" class="mzp-c-hero-cta">
         <div class="show-fxa-supported-signed-out">
-          <a {{ fxa_link_fragment(entrypoint='accounts-page', action='signup', utm_params={'campaign': 'fxa-benefits-page', 'content': 'accounts-page-top-cta', 'medium': 'referral', 'source': 'accounts-page'}) }} class="mzp-c-button mzp-t-primary mzp-t-product" data-button-name="Create account" data-link-type="button" data-cta-position="primary cta" id="features-hero-account">
+          <a
+          {{ fxa_link_fragment(entrypoint='mozilla.org-accounts-page', action='signup', utm_params={'campaign': 'fxa-benefits-page', 'content': 'accounts-page-top-cta', 'medium': 'referral', 'source': 'accounts-page'}) }} class="mzp-c-button mzp-t-primary mzp-t-product" data-cta-position="primary cta" id="features-hero-account">
```

Comment:
Reading the updated issue [description](https://github.com/mozilla/bedrock/issues/7061#issue-431606721), it sounds like this should be `mozilla.org-accounts_page`? (using underscores to separate the point of CTA)